### PR TITLE
Feat: coordinates dropdown

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes the following:
- Refactors datasets dropdown into a reusable `<Dropdown />` component.
- Extracts constants into individual exports 'cause sanity.
- Includes `<CoordinatesDropdown />` in the settings page to set the global coordinates format.
- Formats texts in `<AlertsPosition />` and in the `<Map />` header to display degree coordinates format if selected.

> Run `npm i` before testing.

### CODE REVIEW

File: app/config/constants/index.js [hunk 1]
```diff
@@ -1,81 +1,113 @@
...
-      type: 'Polygon',
-      coordinates: [
-        [
-          [-147.65625, -51.8357775204525],
-          [-147.65625, 70.959697166864],
-          [159.9609375, 70.959697166864],
-          [159.9609375, -51.8357775204525],
-          [-147.65625, -51.8357775204525]
-        ]
+export const APP_NAME = 'Forest Watcher';
```
Comment (j8seangel) [inline app/config/constants/index.js]: I like this a lot! now we can use the constants directly 👏

File: app/offline/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const persistNative = (store, options, callback) => {
       getPersistedStore().purge();
       resetAlertsDb();
     } else {
-      getPersistedStore(); // .purge to clean the offline data
+      getPersistedStore().purge(); // .purge to clean the offline data
```
Comment (j8seangel) [inline app/offline/index.js]: ⚠️ ☠️

Comment (sorodrigo) [inline app/offline/index.js]: Oops 💣

**Comment by sorodrigo** [comment]
This PR now includes a BREAKING CHANGE that depends on this [https://github.com/Vizzuality/react-native-maps/pull/10](url) let's wait for that PR to be merged.

**Pull request merged**